### ISSUE TITLE
Segmentation fault when importing osmium and torch

### ISSUE THREAD

**Issue by SBCV** [osmcode/pyosmium#123]
Running
```
import torch
import osmium
```
causes 

```
Process finished with exit code 139 (interrupted by signal 11: SIGSEGV)
```
I'm running Ubuntu 18.04 with miniconda. 
`pip list` shows me that I'm using osmium 2.15.3 and torch 1.3.1.

**Comment by lonvia** [comment]
This is likely a similar issue like #116 with incompatible libraries. Try installing pyosmium from source with: `pip install --no-binary :all: osmium` (you need to have the prerequisites installed as mentioned in the [README](https://github.com/osmcode/pyosmium#installing-prerequisites)).

**Comment by lonvia** [comment]
https://uwekorn.com/2019/09/15/how-we-build-apache-arrows-manylinux-wheels.html describes the issue in detail and also the solution. It comes down to restricting the exported symbols.

**Comment by lonvia** [comment]
Digging more into this: torch is heavily exporting all kinds of stuff from the stdlibc++, pyosmium doesn't export any symbols. So maybe the symbols are not the issue.

https://github.com/pybind/pybind11/issues/1262  mentions that two modules using pybind11 might clash. For that issue an update to pybind11 2.4 would help. The latest release is still shipped with 2.3.

**Comment by lonvia** [comment]
Fixed with manylinux2010 binary wheels. Please be aware that a recent version of pip is needed. The version shipped with Ubuntu is too old.
